_build/stac.hbs

```diff
@@ -46,11 +46,19 @@
 {{#each Configurations}}
 	{{{copyJson this}}},
 {{/each}}
+{{#if DocumentationLinks }}    
+    {
+      "href": "{{{find DocumentationLinks 'href' 'href'}}}",
+      "rel": "about",
+      "type: "text/html",

```

Comment:
There is a " missing after type leading to the json that is generated not being valid (you can see this when you look at one of the collections in firefox). Chrome somehow is not so strict on the format.